content/blog/2025/logical-options.md

```diff
@@ -0,0 +1,319 @@
+---
+title: How do we move logical shorthands forward?
+sub: There are several proposals, but one major road block
+author: miriam
+date: 2025-03-27
+action:
+  text: Sponsor work on logical shorthands
+  url: https://opencollective.com/oddbird-open-source/contribute/css-logical-shorthands-86141
+summary: >
+  We're trying to make progress
+  on shorthand syntax
+  for CSS logical properties.
+  But the path forward
+  depends on where we hope to be
+  a decade from now.
+---
+
+## What are flow-relative 'logical' properties?
+
+The specification for
+[CSS Logical Properties & Values](https://www.w3.org/TR/css-logical/)
```

Comment:
I like how our ampersand looks, but it stands out that we use `&` a lot more than `and` in this article. Do we have a styleguide for that?

Reply:
I removed it in sentences and left it in titles